### PR TITLE
Convert utilities/values* to typescript

### PR DESCRIPTION
**Pull request by harshkhandeparkar**
## Description
- Converted utilities/values* to ts
- Apparently it isnt used anywhere (¯\\\_(ツ)_/¯)so this part was easy 

## Motivation and Context

[issue](https://github.com/BrainJS/brain.js/issues/551)
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] ¯\\\_(ツ)_/¯

## Author's Checklist:

- [x] My code focuses on the main motivation and avoids scope creep.
- [x] My code passes current tests and adds new tests where possible.
- [ ] My code is [SOLID](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)) and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
- [ ] I have updated the documentation as needed.

## Reviewer's Checklist:
- [ ] I kept my comments to the author positive, specific, and productive.
- [ ] I tested the code and didn't find any new problems.
- [ ] I think the motivation is good for the project.
- [ ] I think the code works to satisfies the motivation.

### CODE REVIEW

File: src/utilities/values-2d.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+/**
+ * Returns a matrix of given width and height with each element filled with the same value
+ */
+export function values2D(
+  width: number,
+  height: number,
+  value: number
+): Float32Array[] {
+  const result = new Array(height);
```
Comment (mubaidr) [inline src/utilities/values-2d.ts]: I think `Float32Array` constructor should be preffered here.

Comment (harshkhandeparkar) [reply src/utilities/values-2d.ts]: I dont that will work because Float32Arrays need to have Float32s inside, not other Float32Arrays.

Comment (mubaidr) [reply src/utilities/values-2d.ts]: return type is `Float32Array[]` and we are returning `result` here which is `new Array(height)`

Comment (harshkhandeparkar) [reply src/utilities/values-2d.ts]: Ohk. I'll make it result: `Float32Array[]` then.

Comment (harshkhandeparkar) [reply src/utilities/values-2d.ts]: Done ig.

**Pull request merged**